app/views/companies/show.html.haml

```diff
@@ -1,3 +1,7 @@
+- content_for :javascript_for_view do
+  = javascript_include_tag "https://maps.googleapis.com/maps/api/js?key=#{Geocoder.config[:api_key]}&language=#{I18n.locale}&region=SE"
+
+
```

Comment:
We don't need this is this file.

We _do_ need it in `payments/create.html.haml` (payment for is not showing up right now as a result).

Reply:
Um, we _do_ need the google maps javascript.  (Perhaps your mind was just so totally on the payments stuff, you misread this?)
But this did prompt me to put scenarios into the feature file.

And I'll add the Hips javascript stuff _is_ loaded into `payments/create.html.haml`

Reply:
Also --

What test (spec or feature) needs to be written so that we are clear on what should happen _and_ are testing for it?

If you give me the general idea, I'll write it.

Reply:
And, my comment should have been `payment form is not showing up right now`.

I need to slow down when I'm entering these comments and confirm they're correct !

Reply:
Good question re testing .... this approach (cucumber) seems like the hard way to conduct the tests _and_ to maintain the tests as we change where JS assets are loaded (and, perhaps in the future, CSS).

Another approach would be to use rspec view tests.  For instance:

`Confirm certains assets NOT loaded on every page`
  .... here, test that these assets are not loaded via `layouts/application.html.haml`.

`Confirm Ckeditor assets loaded on appropriate pages`

etc ....